tests/subsys/storage/stream/stream_flash_nv/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  storage.stream_flash_nv:
+    platform_allow: native_posix native_posix_64
```

Comment:
But the code will actually work on any platform right? In that case we should remove this.

Reply:
The test right now probably wouldn't work on any platform since it writes to offset 0 on the flash device. Is it required to run on any platform?